<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterMosaic</class>
 <widget class="QDialog" name="FilterMosaic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>马赛克</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>模糊半径</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>等比例</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_x">
              <property name="text">
               <string>水平：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSlider_x">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>4</number>
              </property>
              <property name="tracking">
               <bool>true</bool>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_x">
              <property name="accelerated">
               <bool>true</bool>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>4</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_y">
              <property name="text">
               <string>竖直：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSlider_y">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>4</number>
              </property>
              <property name="tracking">
               <bool>true</bool>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_y">
              <property name="accelerated">
               <bool>true</bool>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>4</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="pushButton_reset">
       <property name="text">
        <string>重置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_OK">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider_x</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_x</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_x</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_x</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_y</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_y</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_y</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_y</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_OK</sender>
   <signal>clicked()</signal>
   <receiver>FilterMosaic</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>FilterMosaic</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
